addons/event/views/event_views.xml

```diff
@@ -135,6 +135,23 @@
             </field>
         </record>
 
+        <!-- We need kanban view to be displayed first while coming from registration desk, so we have created
+         new action and changed the view sequence. -->
+        <record id="event_registration_action_kanban" model="ir.actions.act_window">
```

Comment:
I see that you created this new action but... where is it used?

Reply:
Enterprise ... I think it is better to keep it here in case we want to use it someday (and avoid yet another action creation).